tiny_obj_loader.h

```diff
@@ -1855,18 +1884,25 @@ bool LoadObj(attrib_t *attrib, std::vector<shape_t> *shapes,
     // group name
     if (token[0] == 'g' && IS_SPACE((token[1]))) {
```

Comment:
Please see the modifications to the processing of a group. When a group changes, a new shape is added as before.